app/controllers/spotlight/catalog_controller.rb

```diff
@@ -39,7 +39,15 @@ class CatalogController < ::CatalogController
 
     def show
       super
-
+      @docs = []
+      if  @document.sidecars.length>0 && @document.sidecars.first.compound?
+		  @document.sidecars.first.data["configured_fields"]["items"].each do |cur_id|
```

Comment:
I'm not sure `sidecars.first` is what we really want, as `SolrDocument#sidecars` returns all the sidecars for an object across all exhibits. Maybe `@document.sidecar(current_exhibit)` would be better?